server/sonar-server/src/main/java/org/sonar/server/computation/task/step/ComputationStepExecutor.java

```diff
@@ -66,6 +66,16 @@ private void executeSteps(Profiler stepProfiler) {
     }
   }
 
+  private void executeListener(boolean allStepsExecuted) {
+    try {
+      listener.finished(allStepsExecuted);
+    } catch (Throwable e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Catch Exception instead of Throwable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1181)